pkg/cli/admin/inspect/inspect.go

```diff
@@ -243,6 +243,10 @@ func (o *InspectOptions) Run() error {
 		return err
 	}
 
+	if err := InspectDiscovery(context.Background(), o.DestDir, discoveryClient); err != nil {
+		return fmt.Errorf("failed inspecting discovery: %w", err)
```

Comment:
In my opinion, we can continue inspection after accumulating this error to allErrs. Otherwise, any failure would stop the entire inspection.

Reply:
> In my opinion, we can continue inspection after accumulating this error to allErrs. Otherwise, any failure would stop the entire inspection

Agreed.  I thought of this in the shower this morning.